### PR TITLE
Re-revert "Consistent installation title"

### PR DESCRIPTION
**Pull request by Tc-001**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

My second shot at this PR, for context see #2262 and #2265 

The titles now are:
- Installation (Automatic CLI)
- Installation (Manual Setup)

I am using the tab names, so it should still be consistent, they also have around the same breakpoint so the line amount should match on most screen sizes

cc @delucis, let me know if this works for you

### CODE REVIEW

File: src/pages/en/install/auto.mdx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Astro with the Automatic CLI
+title: Installation (Automatic CLI)
```
Comment (delucis) [inline src/pages/en/install/auto.mdx:2]:
```suggestion
title: Install Astro (Automatic CLI)
```

File: src/pages/en/install/manual.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Install Astro manually
+title: Installation (Manual Setup)
```
Comment (delucis) [inline src/pages/en/install/manual.mdx:2]:
```suggestion
title: Install Astro (Manual Setup)
```

**Review by delucis — COMMENTED** [review]
Thanks for circling back to this @Tc-001!

What do you think of this suggestion preserving “Install Astro” instead of “Installation”? I kind of like having “Astro” in there for clarity in search engine results and this is only very slightly longer than your suggestion.

**Comment by Tc-001** [comment]
I like it!
Will make the change in a few hrs

**Comment by sarah11918** [comment]
@Tc-001 - just pinging for freshness! Make Chris' changes, and it's good to merge, I think!

**Comment by Tc-001** [comment]
They are a bit too different for the effect, but the following works:

- Install Astro with manual setup
- Install Astro with the Automatic CLI

**Comment by sarah11918** [comment]
What does it look like if you compare:

> Install Astro Wizard

> Install Astro manually

I actually *like* how Install Auto manually looks as a title. So, can we tighten up the Auto? Other suggestions for phrasing?

**Comment by Tc-001** [comment]
|a|b|c|
|---|---|---|
|image| image | image|

"Install Astro automatically" just barely is smaller than the word wrap point, but unsure how it looks on other browsers
Wizard works but I think the wording is a bit weird? Will see if I can come up with anything

**Comment by sarah11918** [comment]
Install Astro via Wizard  ??

Install Astro with Wizard ??

**Comment by Tc-001** [comment]
Maybe? Still a bit strange for my taste, but they do fit!

**Comment by Tc-001** [comment]
I have found some, but needing them to be around the same length is a bit limiting, I don't think there is a perfect one

**Comment by Tc-001** [comment]
For me personally [this](https://github.com/withastro/docs/pull/2326#issuecomment-1377828690) makes the most sense, as it is a line and a bit in length in the largest size

But maybe it would be better to remove the tabs and make them distinctively 2 different pages? They could link to each other with something like:
> "The wizard will now run some commands, if you want to know what they are or run them yourself, click here. "

**Comment by Tc-001** [comment]
I think I will sleep on it, is is getting quite late
Also will probably move to discord, a lot easier to paste screenshots there

Best of lucks to your projects!

**Comment by kevinzunigacuellar** [comment]
Is there any reason to keep the automatic in "automatic cli"?
 I think "with the cli" is simpler

**Comment by Tc-001** [comment]
It could work and fits in one line!
Although it wraps a lot sooner so the effect would break if the browser is not maximized, but it would do it's job ~80% of the time

**Comment by sarah11918** [comment]
Coming back to this.... 😅 

What about some distinction more like:

"Automatic Setup" / "Install Manually"

- They *both* involve the CLI, just in one case you're installing and running the wizard, and in the other, you're just installing Astro. So, I don't think using "CLI" for one vs the other is exactly correct?
- It's not the "installing" that's automatic, it's the SETUP with all the dependencies, eh?

**Comment by sarah11918** [comment]
OK! So we have a different fix for this issue.... we're going to tackle this in #2390 and kill off the fake tabs entirely!  :hocho: 

This is the only time we use tabs to "fake" page navigation, and it's a pattern that is clearly problematic and that we shouldn't enable. Plus, having two separate pages allows us to customize the top of the page so we can still nudge people to `create astro`.  

Since the whole reason for this pattern was to avoid duplicating content, but now we actually want that content to be different anyway, I'm closing this issue in favour of the solution to making two entirely different pages. There's no need for this content to be this closely aligned anymore.

**Pull request closed without merging**